src/middlewared/middlewared/plugins/keychain.py

```diff
@@ -703,7 +703,10 @@ def ssh_pair(self, data):
         os.chown(dotsshdir, user["uid"], user["group"]["bsdgrp_gid"])
 
         # Write public key in user authorized_keys for SSH
-        with open(f"{dotsshdir}/authorized_keys", "a+") as f:
+        authorized_keys_path = f"{dotsshdir}/authorized_keys"
```

Comment:
Any reason why you moved the path into its own variable? Seems like that change is superfluous.